README.md

```diff
@@ -0,0 +1,7 @@
+# Java Core June 2021
+
+## *Emelianchik Svetlana*
+
+| Number | Solution  | Short description
+| --- | --- | --- |
+| HW1 | [Console printer](https://github.com/NikolaevArtem/Java_Core_June_2021/tree/feature/SvetlanaEmelianchik/src/main/java/homework_1) | The app that reads input arguments and prints them, until "error" argument |
```

Comment:
It still looks at my PR

Reply:
The link leads to my branch with homework_1 package. Is it not correct?

Reply:
My bad, i looked at the wrong place. You're right, it works fine